.github/workflows/cargo-test.yaml

```diff
@@ -39,3 +44,31 @@ jobs:
         with:
           command: test
           args: --manifest-path ./actix-api/Cargo.toml
+
+
+  test-ui:
+    name: UI Test Suite
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+        with:
+          working-directory: ./yew-ui
+
+      - name: Install nightly toolchain and wasm32-unknown-unknown
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          target: wasm32-unknown-unknown
+          toolchain: nightly-2022-10-21
+          override: true
+
+      - name: Install wasm-bindgen-cli
+        uses: jetli/wasm-bindgen-action@v0.2.0
+        with:
+          version: 'latest'
+
+      - name: Run cargo test
+        run: |
+          cd yew-ui
+          cargo test
```

Comment:
I think that there's a tool called `wasm-pack` that is used to test browser tests

Reply:
Yeah it's installed in the step before this with wasm-bindgen